pkg/aspectsupport/accessLogger/config/config.proto

```diff
@@ -0,0 +1,54 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package pkg.adaptersupport.accessLogger.config;
+
+option go_package="config";
+
+// Configures an individual logger aspect implementation.
+//
+// Example usage:
+// kind: istio/accessLogger
+// params:
+//    log_name: "logs/access_log"
+//    attributes: "source.ip","host.URL","
+message Params {
+    // Provides a way to identify a collection of related log entries.
+    string log_name = 1;
+
+    // A list of names of attributes that will be used to generate
+    // access log entries.
+    repeated string attributes = 2;
+
+    // This template will be applied to the map of attributes names to values
+    // generated by the mixer from the specified set of attributes
+    // to create a textual access log entry.
+    //
+    // The template MUST be specified in a format compatible with:
```

Comment:
I totally understand. At the moment, however, I'm not sure how to best "hide" this bit.

We'd need something that maps to/from attribute values to [Custom Log Formats](https://httpd.apache.org/docs/1.3/mod/mod_log_config.html#formats) (or similar) to allow the operator to specify in the language they are more familiar with (something like: `"%h %l %u %t \"%r\" %>s %b"`).

If you have a clever approach for that, I'd be very interested.

Perhaps we could maintain a mapping in the manager (once we have a schema for well-known attributes)?